src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,94 @@
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+
+import { Person } from '../../types/Person';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  const findParent = (parentName: string | null) => {
```

Comment:
It's not always `parent`, it could be the current person, so I'd name it `findPerson`
![image](https://user-images.githubusercontent.com/86923089/212533929-22655a2e-a5ed-473f-ba30-3c52cb52cf65.png)

Reply:
Also, this function returns valid `JSX`, so you overwrite it as component `PersonLink`